.vscode/launch.json

```diff
@@ -8,8 +8,17 @@
             "type": "node",
             "request": "launch",
             "name": "Launch Program",
-            "program": "${workspaceRoot}/lib/dataValidator.js",
-            "cwd": "${workspaceRoot}"
+            "program": "${workspaceRoot}/cli.js",
+            "cwd": "${workspaceRoot}",
+            "args": [
+                "validate-example",
+                "..\\upstream\\azure-rest-api-specs\\arm-customer-insights\\2016-01-01\\swagger\\customer-insights.json",
```

Comment:
minor: it'd be good if you can point to the remote github version of this file so that we don't have to change it.